packages/EasyErrorHandler/src/Bridge/Laravel/Provider/EasyErrorHandlerServiceProvider.php

```diff
@@ -197,18 +198,27 @@ static function (Container $app): BugsnagReporterProvider {
         }
 
         $this->app->singleton(
-            ErrorCodesProviderInterface::class,
+            ErrorCodesFromInterfaceProvider::class,
             static function (): ErrorCodesProviderInterface {
-                return new ErrorCodesProvider(\config('easy-error-handler.error_codes_interface'));
+                return new ErrorCodesFromInterfaceProvider(\config('easy-error-handler.error_codes_interface'));
             }
         );
+        $this->app->tag(ErrorCodesFromInterfaceProvider::class, [BridgeConstantsInterface::TAG_ERROR_CODES_PROVIDER]);
+
+        $this->app->singleton(
+            ErrorCodesFromEnumProvider::class,
+            function (): ErrorCodesProviderInterface {
+                return new ErrorCodesFromEnumProvider($this->app->basePath() . '/app');
```

Comment:
Instead of using `$this->app` please use a static function and expect an argument `Container $app`.

Reply:
`Container` interface is not enough for `phpstan`. `Application $app` added. Wdyt?